Solution1/Solution1.Tests/ConsoleApp/Extensions/HttpStatusCodeExtensionsTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ConsoleApp.Extensions;
+using System.Net;
+using Xunit;
+
+namespace Weather.Tests.ConsoleApp.Extensions
+{
+    
+    public class HttpStatusCodeExtensionsTests
+    {
+        [Fact]
+        public void GetStringRepresentation_GetStringRepresentationFromHttpStatusCode_Success()
+        {
+            // Arrange
+            HttpStatusCode? httpStatusCode = HttpStatusCode.BadRequest;
```

Comment:
why you make it nullable?? and why you not using var?

Reply:
+